name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/traderepublic/TradeRepublicPDFExtractorTest.java

```diff
@@ -1155,6 +1156,246 @@ public void testKontoauszug05()
                         hasSource("Kontoauszug05.txt"), hasNote(null))));
     }
 
+    @Test
+    public void testKontoauszug06()
+    {
+        TradeRepublicPDFExtractor extractor = new TradeRepublicPDFExtractor(new Client());
+
+        List<Exception> errors = new ArrayList<>();
+
+        List<Item> results = extractor.extract(PDFInputFile.loadTestCase(getClass(), "Kontoauszug06.txt"),
+                        errors);
+
+        assertThat(errors, empty());
+        assertThat(countSecurities(results), is(0L));
+        assertThat(countBuySell(results), is(0L));
+        assertThat(countAccountTransactions(results), is(20L));
+        assertThat(results.size(), is(20));
+        new AssertImportActions().check(results, CurrencyUnit.EUR);
+
+        // assert transaction
+        assertThat(results, hasItem(interest(hasDate("2024-04-01"), hasAmount("EUR", 147.34), //
```

Comment:
I would mark this as NonImportable. We don't know the taxes and fee situation.

Reply:
Why? Are they not interest payments? (Interest on the assets?)

Reply:
Yes, but we don't know if taxes were applied and manipulate the payed taxes overview. I could confirm this in Dec 2023. Freibetrag was exhausted and taxes were applied. No hint about taxes was visible in the export.

Reply:
We need a pdf debug to check this

Reply:
I can't provide a debug of something which is not contained. You can see in both exports an interest booking. Both do not contain taxes.